README.md

```diff
@@ -1,21 +1,24 @@
 # Capstone_Appointment_back_end
 
-> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface. This app uses RoR, RSPec, Capybara, Devise, and CanCanCan.
+> This application is developed using Ruby on Rails(RoR) CRUD capabilities. It allows users to create motorcycles and appointments to purchase and post them in a web interface.
 
 ## Built With
 
 - Ruby On Rails (RoR)
 - PostegreSQL
 - RSpec
-- Capybara
-- Devise
-- CanCanCan
-- Bullet
+- Rswag
+- jwt
+- Bcrypt
+- rack-cors and
+- image-processing
 
 ## Live Demo (if available)
 
 [Live Demo Link]()
-
+[API Documentation](https://motocycle-booking.herokuapp.com/api-docs/)
+[Front-end link](https://final-capstone-project-udqc.vercel.app/splash_page)
+ 
```

Comment:
- [ ] On your testing instructions, if I follow your instructions, I get this:

![image](https://user-images.githubusercontent.com/57421823/199777774-c77dedc8-ea23-46cd-8d43-c4c6f1c3be79.png)

## Explanations

This is because those 14 failures are swagger tests.  But we don't want any person (e.g. recruiter) to get any failed examples in your project.

## Tips and extra help

Instead of instructing your users to just run `rspec --format documentation`, tell them to run specific folders with the correct tool:

### First, normal rspec for your non-rswag tests  (in this case you have tests for models):

`rspec spec/models --format documentation`:

Result:
![image](https://user-images.githubusercontent.com/57421823/199778752-7e0d162e-a8fa-45b0-b964-367e89f4a215.png)

### After that, tell them to run the swagger tests:

`rake rswag`

Result:
![image](https://user-images.githubusercontent.com/57421823/199779322-484e9c00-1de5-4685-8daa-21b8c53af148.png)

### So in your readme instructions to run tests:

```bash
rspec spec/models --format documentation
rake rswag
```

That way, nobody will ever get a failed example, all the tests will always be green 💪